CHANGELOG.next.asciidoc

```diff
@@ -65,6 +65,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add `while_pattern` type to multiline reader. {pull}19662[19662]
 - Tracking session end reason in panw module. {pull}18705[18705]
 - Fix PANW field spelling "veredict" to "verdict" on event.action {pull}18808[18808]
+- Adding fix to checkpoint module when logs contain time field. {pull}20567[20567]
```

Comment:
Please, could you move the changelog entry into de `bugfixes` section? now it is under the breaking changes one.